src/NosCore.Controllers/Warehouse/DepositPacketHandler.cs

```diff
@@ -36,12 +37,13 @@ public DepositPacketHandler(IWarehouseHttpClient warehouseHttpClient)
             _warehouseHttpClient = warehouseHttpClient;
         }
 
-        public override void Execute(DepositPacket depositPacket, ClientSession clientSession)
+        public override Task Execute(DepositPacket depositPacket, ClientSession clientSession)
         {
-            IItemInstance itemInstance = null;
+            IItemInstance itemInstance = new ItemInstance();
             short slot = 0;
-            var warehouseItems = _warehouseHttpClient.DepositItem(clientSession.Character.CharacterId,
+            var warehouseItems = _warehouseHttpClient.DepositItem(clientSession.Character!.CharacterId,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused local variable 'warehouseItems'.](https://app.codacy.com/manual/NosCoreIO/NosCore/pullRequest?prid=5262030)